src/Platform/Microsoft.Testing.Platform/Services/TestApplicationResult.cs

```diff
@@ -17,7 +18,17 @@ internal sealed class TestApplicationResult : ITestApplicationProcessExitCode, I
     private readonly ICommandLineOptions _commandLineOptions;
     private readonly IEnvironment _environment;
     private readonly IStopPoliciesService _policiesService;
+    private readonly IPlatformOpenTelemetryService? _otelService;
+    private readonly ICounter<int>? _totalDiscoveredTests;
+    private readonly ICounter<int>? _totalStartedTests;
+    private readonly ICounter<int>? _totalCompletedTests;
+    private readonly ICounter<int>? _totalPassedTests;
+    private readonly ICounter<int>? _totalFailedTests;
+    private readonly ICounter<int>? _totalSkippedTests;
+    private readonly ICounter<int>? _totalUnknownedTests;
+    private readonly IHistogram<double>? _totalDuration;
```

Comment:
Are we considering parallelization here? Not sure how this all ends up being shown.

Reply:
It was working on my small manual tests